### PR TITLE
Calling Cards Blog Post

### PR DESCRIPTION
**Pull request by cmatKhan**
This is the much anticipated Calling Cards blog post.

I also adjusted the README instructions on launching the dev server to more explicitly describe the procedure I followed -- that isn't related to the calling cards blog and might be better done in a different push.

### CODE REVIEW

**Review by mashehu — APPROVED** [review]
Great job!
just some smaller things (commit with fixes for the figures is incoming).

File: README.md [hunk 1]
```diff
@@ -39,7 +39,14 @@ npm install
...
+`cd` into the `sites` directory on which you would like to develop, for example:
+
+```bash
+cd sites/main-site
+```
+
+Then execute the following command:
 
 ```bash
 npm run dev
```
[the remark this answers is not in the document]
Comment (cmatKhan) [reply README.md]: I removed this commit from this branch

File: sites/main-site/src/assets/images/blog/callingcards/callingcards_vs_chip.png
[the remark this answers is not in the document]
Comment (cmatKhan) [reply sites/main-site/src/assets/images/blog/callingcards/callingcards_vs_chip.png:1]: done in cb8fe6a71d67a337cd062541cd9c78f249220492

File: sites/main-site/src/content/blog/2024/callingcards_intro.mdx [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+the DNA binding locations of proteins called Transcription Factors. Transcription
+Factors (TF) are a class of proteins capable of binding to DNA to affect the
+transcription of genes, typically those nearest the binding event. In order to record
+where a given TF binds, calling cards harnesses the power of a retrotransposon, or a
+transcribed genomic sequence capable of copying itself back into the
+genome, to deposit a molecular barcode into the genome when a TF binds.
+
+# Calling Cards in Yeast
+
+<div style={{ textAlign: 'center', margin: '20px 0' }}>
```
[the remark this answers is not in the document]
Comment (cmatKhan) [reply sites/main-site/src/content/blog/2024/callingcards_intro.mdx]: ah, right...that is much better. Thank you for changing all of those

**Comment by cmatKhan** [comment]
@mashehu thank you for your commits and review.

I removed the README update from this branch, updated the background, accepted your commits and made some minor text edits.

I'm failing the playwright check now -- I'm running this on my local and it seems like it is failing on pages other than the one i updated.

**Comment by mashehu** [comment]
Yes, I hoped the playwright tests would work better with the new monorepo structure, but we haven't found a good way to handle the multiple deployment previews. The tests can be ignored for now.

**Review by mashehu — APPROVED** [review]

File: sites/main-site/src/content/blog/2024/callingcards_intro.mdx [hunk 3]
```diff
@@ -0,0 +1,144 @@
+---
+title: 'nf-core/callingcards - first release!'
+subtitle: 'The nf-core callingcards workflow'
+headerImage: '/assets/images/blog/callingcards/callingcards_metromap.png'
+headerImageAlt: 'Subway diagram of the calingcards pipeline'
+pubDate: 2024-07-10T00:19:00+01:00
```
Comment (mashehu) [inline sites/main-site/src/content/blog/2024/callingcards_intro.mdx:6]:
```suggestion
pubDate: 2024-07-17T00:19:00+01:00
```

**Comment by mashehu** [comment]
okay, good to merge whenever you are ready 🙂

**Comment by FriederikeHanssen** [comment]
The deploy preview looks off (tested on chrome and safari). Is this anythign to be concerned about @mashehu 
Screenshot 2024-07-17 at 11 33 42

**Comment by mashehu** [comment]
Will be fine once it's merged (bug with the new setup and different sources for components)

**Comment by mashehu** [comment]
@cmatKhan I set the `pubDate` now to July 18th midnight. feel free to change that to a date more suitable for you (just want to have the "new blog post" indicator appear for this post as well (stays there for 24h past release)

**Pull request merged**